jcl/src/java.base/share/classes/com/ibm/oti/util/Util.java

```diff
@@ -283,25 +283,26 @@ public static void appendLnTo(Appendable buf) {
 /**
  * Print a stack frame without allocating memory directly.
  * @param e StackTraceElement to add
- * @param elementSource location of the source file.  May be null.
+ * @param elementSource location of the source file. May be null.
  * @param buf Receiver for the text
- * @param extended add additional information
  */
-public static void printStackTraceElement(StackTraceElement e, Object elementSource, Appendable buf, boolean extended) {
+public static void printStackTraceElement(StackTraceElement e, Object elementSource, Appendable buf) {
 	/*[IF Sidecar19-SE]*/
-	final java.lang.String modName = e.getModuleName();
+	String classLoaderName = e.getClassLoaderName();
+	if (null != classLoaderName) {
+		appendTo(buf, classLoaderName);
```

Comment:
I don't think this is overall correct. It may print more information for application modules, but not for modules like java.base

Reply:
> I don't think this is overall correct. It may print more information for application modules, but not for modules like java.base

That is the intent: the classloader name is not printed for `java.base`:

```
"Reference Handler" #2 daemon prio=10 os_prio=31 cpu=0.15ms elapsed=13.16s tid=0x00007fe5f4073800 nid=0x3303 waiting on condition  [0x0000700008646000]
   java.lang.Thread.State: RUNNABLE
	at java.lang.ref.Reference.waitForReferencePendingList(java.base@11/Native Method)
	at java.lang.ref.Reference.processPendingReferences(java.base@11/Reference.java:241)
	at java.lang.ref.Reference$ReferenceHandler.run(java.base@11/Reference.java:213)
```